src/comm/MockLink.General.MetaData.json

```diff
@@ -4,5 +4,7 @@
     "modelName":                    "QGC MockLink Model",
     "firmwareVersion":              "1.0.0",
     "hardwareVersion":              "1.0.0",
-    "supportedCompMetadataTypes":   [ 0, 1 ]
-} 
\ No newline at end of file
+	"metadataTypes": [
+		{"type": 1, "uri": "mftp://[;comp=1]parameter.json.xz", "fileCrc": 1}
```

Comment:
`"fileCrc": 1` - I assume this is because crc checking isn't implemented yet right? I could seem to find it.